.fvm/versions/3.29.2

```diff
@@ -0,0 +1 @@
+C:/Users/dario/fvm/versions/3.29.2
```

Comment:
I doubt that FVM support is necessary for this project. It can easily be installed by users locally if they want it. Also, `C:/Users/dario/fvm/versions/3.29.2` path is only specific to your setup.